rfcs/036-holdings-records/README.md

```diff
@@ -0,0 +1,261 @@
+
+    *   A digital location using field 856 if the location in fixed field 40 is `elro` (online)
+    *   A physical location using fixed field 40 for the location type/label, and 949 for the shelfmark.
+
+*   For now, we won't add any identifiers.
+    We could add them as h-prefixed Sierra IDs, but I don't see much use of those identifiers -- usually people use the b number when discussing the associated holdings record.
+
+### Possible future enhancements
+
+Although we're treating items and holdings as separate types, we might be able to do some de-duplication of data in a future release (e.g. two items *Vol. 1* and *Vol. 2*, and a holdings record *Vols. 1 and 2*).
+This would be quite expensive to do, so we're not going to spend significant time on that until the One Website Project is finished.
+There are better places that time/effort can be spent.
+
+
+
+## Examples
+
+<details>
+<summary>A bib with a single physical holdings record</summary>
+
+b10032538 (<a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1003253?marcData=Y&lang=eng">Library</a>/<a href="https://api.wellcomecollection.org/catalogue/v2/works/esstapc9?include=items">API</a>)
+
+```json
+{
+  "type": "Work",
+  "id": "esstapc9",
+  "holdings": [
+    {
+      "identifiers": [
+        {
+          "identifierType": {"id": "sierra-system-number", "label": "Sierra system number", "type": "IdentifierType"},
+          "value": "1063290"
+        }
+      ]
+      "title": "Vol. 1-7",
+      "locations": [
+        {
+          "locationType": {
+            "id": "closed-stores",
+            "label": "Closed stores",
+            "type": "LocationType"
+          },
+          "label": "Closed stores",
+          "type": "Location"
+        }
+      ],
+      "type": "Holdings"
+    }
+  ],
+  …
+}
+```
+
+</details>
+
+<details>
+<summary>A bib with a single electronic holdings record</summary>
+
+b10035370 (<a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1003537?lang=eng">Library</a>/<a href="https://api.wellcomecollection.org/catalogue/v2/works/fjuk8v9k?include=items">API</a>)
+
+```json
+{
+  "type": "Work",
+  "id": "fjuk8v9k",
+  "holdings": [
+    {
+      "locations": [
+        {
+          "locationType": {
+            "id": "online-resource",
+            "label": "Online resource",
+            "type": "LocationType"
+          },
+          "accessConditions": [
+            {
+              "accessStatus": {
+                "id": "licensed-resources",
+                "label": "Licensed resources",
+                "type": "AccessStatus"
+              },
+              "type": "AccessCondition"
+            }
+          ],
+          "linkText": "View this e-book",
+          "url": "http://ark.cdlib.org/ark:/13030/ft0d5n99m0/",
+          "type": "Location"
+        }
+      ],
+      "type": "Holdings"
+    }
+  ],
+  …
+}
+```
+
+</details>
+
+<details>
+<summary>A bib with multiple physical holdings records</summary>
+
```

Comment:
I need to read whole pull requests before I add comments.... It's interesting here that there is very little data on each holding. Is that the norm, ie are things like enumeration quite uncommon? Again I do wonder about the utility of including the locations here, as surely they are on the items and it's redundant. Ie can we keep holdings as a summary of, um, holdings and then items as the sole locatable object.